File: codigos_de_javascript/arregloIns.js
let arreglo = [24, 2, 11, 6, 3];
let i, j, aux;

console.log("Antes de ordenar:");
console.log(arreglo.join(""));

for (i = 1; i < arreglo.length; i++) {
    aux = arreglo[i];
    j = i - 1;
    while (j >= 0 && arreglo[j] > aux) {
        arreglo[j + 1] = arreglo[j];
        j--;
    }
    arreglo[j + 1] = aux;
}

console.log("Despues de ordenar:");
console.log(arreglo.join(""));

File: codigos_de_python/ArregloIns.py
arreglo = [24, 2, 11, 6, 3]

print("Antes de ordenar:")
print("".join(str(x) for x in arreglo))

# Algoritmo de inserción
for i in range(1, len(arreglo)):
    aux = arreglo[i]
    j = i - 1
    while j >= 0 and arreglo[j] > aux:
        arreglo[j + 1] = arreglo[j]
        j -= 1
    arreglo[j + 1] = aux

print("Despues de ordenar:")
print("".join(str(x) for x in arreglo))
